osu.Game.Rulesets.Osu.Tests/TestSceneSpinnerSpunOut.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using osu.Game.Rulesets.Osu.Mods;
+
+namespace osu.Game.Rulesets.Osu.Tests
+{
+    [TestFixture]
+    public class TestSceneSpinnerSpunOut : TestSceneSpinner
```

Comment:
This test scene is broken because the way it's written, the `[SetUp]` necessary for the tests to even make sense is the *last* step.

Reply:
Also it should actually be *asserting* something.

Reply:
AFAIK the issue of `[SetUp]` being the last step in test browser is also present in `TestSceneSpinnerHidden`.

Reply:
Possible fallacies of pre-existing code are no excuse for fallacies of new code.